Osiris/Config.h

```diff
@@ -9,6 +9,8 @@
 #include "nSkinz/config_.hpp"
 #include "ConfigStructs.h"
 
+#include "SDK/Vector.h"
+
```

Comment:
Insert 
`#include "../../Osiris-master/Osiris/ConfigStructs.h"

#include "../../Osiris-master/Osiris/ConfigStructs.h"`

to avoid an undefined variable